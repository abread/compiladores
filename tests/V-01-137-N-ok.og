auto f(int a) {
  if a then return 5, 61.0, "string1";
  else      return 6, 62.0, "string2";
}
public int og() {
  auto a, b, c = f(1);
  write a;
  write b;
  write c;
  return 0;
}

