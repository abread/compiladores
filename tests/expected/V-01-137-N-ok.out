56.1E1string1
